tasks/task1.js

```diff
@@ -0,0 +1,96 @@
+// Task 1
+function summ(...args) {
+  let sum = 0;
+  for (let i = 0; i < args.length; i += 1) {
+    if (+args[i] && typeof args[i] !== 'boolean') {
+      sum += +args[i];
+    }
```

Comment:
Try to simplify (less code)

Reply:
I tried using reduce. But there was no less code.

1st option:
function summ(...args) {
  if (args.length === 0) return null;
  let sum = 0;
  for (let i = 0; i < args.length; i += 1) {
    if (+args[i] && typeof args[i] !== 'boolean') {
      sum += +args[i];
    }
    return sum;
  }

2nd option:
  function summ(...args) {
    if (args.length === 0) return null;
    const sum = args.reduce((accumulator, item) => {
      if (+item && typeof item !== 'boolean') {
        accumulator += +item;
      }
      return accumulator;
    }, 0);
    return sum;
  }

Reply:
3th option:
shorter but not sure which is better:
function summ2(...args) {
  if (args.length === 0) return null;
  let sum = 0;
  args.forEach(item => {
    sum += (+item && typeof item !== 'boolean') ? +item : 0;
  });
  return sum;
}